<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>700</width><height>420</height></rect>
  </property>
  <property name="windowTitle"><string>TCP Queue Client</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupNetwork">
      <property name="title"><string>Сетевые настройки</string></property>
      <layout class="QHBoxLayout" name="networkLayout">
       <item><widget class="QLabel" name="labelIP"><property name="text"><string>IP:</string></property></widget></item>
       <item><widget class="QLineEdit" name="lineEditIP"><property name="text"><string>127.0.0.1</string></property></widget></item>
       <item><widget class="QLabel" name="labelPort"><property name="text"><string>Порт:</string></property></widget></item>
       <item><widget class="QLineEdit" name="lineEditPort"><property name="text"><string>12345</string></property></widget></item>
       <item><widget class="QPushButton" name="btnConnect"><property name="text"><string>Подключиться</string></property></widget></item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupQueue">
      <property name="title"><string>Работа с очередью</string></property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0"><widget class="QLabel" name="labelInput"><property name="text"><string>Массив чисел:</string></property></widget></item>
       <item row="0" column="1" colspan="2"><widget class="QLineEdit" name="lineEditInput"><property name="placeholderText"><string>Например: 5, 2, 8, 1</string></property></widget></item>
       <item row="1" column="0"><widget class="QPushButton" name="btnAddToQueue"><property name="text"><string>Добавить в очередь</string></property></widget></item>
       <item row="1" column="1"><widget class="QPushButton" name="btnRequestQueue"><property name="text"><string>Запросить данные</string></property></widget></item>
       <item row="1" column="2"><widget class="QPushButton" name="btnClearQueue"><property name="text"><string>Очистить очередь</string></property></widget></item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupOutput">
      <property name="title"><string>Результат</string></property>
      <layout class="QVBoxLayout" name="outputLayout">
       <item><widget class="QTextEdit" name="textEditOutput"><property name="readOnly"><bool>true</bool></property></widget></item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
